bundles/org.openhab.binding.hue/src/main/resources/OH-INF/thing/Device.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hue" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- CLIP 2 device thing -->
+	<thing-type id="device">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="clip2"/>
+		</supported-bridge-type-refs>
+
+		<label>Hue Device</label>
+		<description>A Hue API v2 device with channels depending on its actual capabilities.</description>
+
+		<channels>
+			<channel id="brightness" typeId="system.brightness"/>
+			<channel id="color" typeId="system.color"/>
+			<channel id="color_temperature" typeId="system.color-temperature"/>
```

Comment:
Since we have a clean slate here, I would suggest to follow naming convention for channels, i.e. colorTemperature. We do not have to consider backwards compatibility since we have new bridge and thing types.

Reply:
Ok.

Reply:
.. but note that I had to rename the constants in HueBindingConstants in order to avoid naming conflicts..